rpc/src/lib.rs

```diff
@@ -104,11 +101,9 @@ where
 		+ 'static,
 	C::Api: substrate_frame_rpc_system::AccountNonceApi<Block, AccountId, Nonce>,
 	C::Api: pallet_transaction_payment_rpc::TransactionPaymentRuntimeApi<Block, Balance>,
-	C::Api: pallet_contracts_rpc::ContractsRuntimeApi<Block, AccountId, Balance, BlockNumber, Hash>,
 	C::Api: BlockBuilder<Block>,
 	P: TransactionPool + Sync + Send + 'static,
 {
-	use pallet_contracts_rpc::{Contracts, ContractsApiServer};
```

Comment:
Why remove pallet contract's RPC APIs ?

Reply:
Because this has been removed in Polkadot-v0.9.31 https://github.com/paritytech/substrate/pull/12358